server/routerlicious/packages/services-utils/src/morganLoggerMiddleware.ts

```diff
@@ -45,10 +45,10 @@ export function jsonMorganLoggerMiddleware(
                 additionalProperties = computeAdditionalProperties(tokens, req, res);
             }
             const properties = {
-                [HttpProperties.method]: tokens.method(req, res),
+                [HttpProperties.method]: tokens.method(req, res) ?? "METHOD_UNAVAILABLE",
```

Comment:
>[HttpProperties.method]: tokens.method(req, res) ?? "METHOD_UNAVAILABLE", [](http://example.com/codeflow?start=16&length=73)

tokens.method( ) just returns req.method. Wondered how req.method wouldn't be populated...

Reply:
I think it will always be, but just in case, since Method is one of those fields we use in our metrics, I decided to add that here just as a fallback in case it can be empty - so that we avoid having missing datapoints.